src/components/Filters/Filters.tsx

```diff
@@ -1,22 +1,39 @@
-import React, { useState, useRef } from 'react';
+import React, { useState } from 'react';
 import styles from './Filters.module.css';
 import Button from '../Button';
-import { CheckboxField, FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
+import { FiltersCheckboxField } from '../FiltersCheckboxField/FiltersCheckboxField';
+import { FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
 import Icon from '../Icon';
 
 export type Props = {
   /**
-   * Checkbox fields that will be displayed in the filters drawer.
+   * Text to be placed in the button that activates the Filters Drawer
    */
-  checkboxFields: CheckboxField[];
+  buttonText?: string;
```

Comment:
I had to go look at the stories to be sure what this prop did because I was thinking it might be for the primary button inside the drawer. Maybe we could make it more specific like `triggerButtonText` or even just `triggerText`?

Reply:
`triggerText` works, prop concise yet informative prop names are always great :D